project/Dependencies.scala

```diff
@@ -69,6 +69,12 @@ object Dependencies {
   private[this] object pagopa {
     lazy val namespace = "it.pagopa"
 
+    lazy val attributeRegistryManagementClient =
+      namespace %% "interop-be-attribute-registry-management-client" % attributeRegistryManagementVersion
```

Comment:
not required, in this PR we are reading from the read model and not calling the management
```suggestion
```